.travis.yml

```diff
@@ -32,20 +32,43 @@ branches:
 
 matrix:
   include:
-    - env: RUN="vet fmt migrations integration godep-restore errcheck generate dashlint rpm"
+    #
+    # Current Go version build tasks:
+    #
+    - env: GO_VERSION="1.10" RUN="vet fmt migrations integration godep-restore errcheck generate dashlint rpm"
     # Config changes that have landed in master but not yet been applied to
     # production can be made in boulder-config-next.json.
-    - env: RUN="integration" BOULDER_CONFIG_DIR="test/config-next"
-    - env: RUN="unit"
-    - env: RUN="unit-next" BOULDER_CONFIG_DIR="test/config-next"
-    - env: RUN="coverage"
+    - env: GO_VERSION="1.10" RUN="integration" BOULDER_CONFIG_DIR="test/config-next"
+    - env: GO_VERSION="1.10" RUN="unit"
+    - env: GO_VERSION="1.10" RUN="unit-next" BOULDER_CONFIG_DIR="test/config-next"
+    - env: GO_VERSION="1.10" RUN="coverage"
+
+    #
+    # Next Go version build tasks:
+    #
+    - env: GO_VERSION="1.10.1" RUN="vet fmt migrations integration godep-restore errcheck generate dashlint rpm"
+    # Config changes that have landed in master but not yet been applied to
+    # production can be made in boulder-config-next.json.
+    - env: GO_VERSION="1.10.1" RUN="integration" BOULDER_CONFIG_DIR="test/config-next"
+    - env: GO_VERSION="1.10.1" RUN="unit"
+    - env: GO_VERSION="1.10.1" RUN="unit-next" BOULDER_CONFIG_DIR="test/config-next"
   fast_finish: true
   allow_failures:
     - env: RUN="coverage"
 
+# We require a newer version of docker-compose than is installed by way of the
+# "services: docker" directive. Per the travis docs[0] this is best remedied
+# with the following `before_install` commands.
+# [0]: https://docs.travis-ci.com/user/docker/#Using-Docker-Compose
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
```

Comment:
We may never be able to get rid of `sudo: required`, but just in case: It would be a little nicer to fetch the docker-compose binary into a local file owned by non-root, and execute it by its full path name (e.g. `$HOME/docker-compose`).

Reply:
:+1: addressed in 23214c4